docker/manage

```diff
@@ -145,9 +145,13 @@ configureEnvironment() {
     esac
   done
 
+  if [ ! -f ../oidc-controller/signing_key.pem ]; then
+    openssl genrsa -out ../oidc-controller/signing_key.pem 2048
+  fi
+
   ## global
   export COMPOSE_PROJECT_NAME="${COMPOSE_PROJECT_NAME:-vc-authn}"
-  export GENESIS_URL="${GENESIS_URL:-http://$DOCKERHOST:9000/genesis}"
+  export GENESIS_URL="${GENESIS_URL:-http://$DOCKERHOST:9001/genesis}"
```

Comment:
This will mess-up "normal" deployments that rely on `von-netowrk` running on port `9000`. The change should be reverted - for your machine you should be able to launch the project using `GENESIS_URL=http://host.docker.internal:9001/genesis ./manage start` instead

Reply:
Good catch, yes this was not supposed to be committed.